### DISCUSSION TITLE
Confirmation Button on Host print Complete

### DISCUSSION THREAD

**Discussion by aryajuanda** [Jyers/Marlin#421] [Ideas]
when finishing printing from octoprint, LCD will display Confirmation Button, i think it will be better to display main menu after printing job finished without confirmation button.. because with M73 and display layer progress it show not 100% progress bar with confirmation button in my setup..

![IMG_3248](https://user-images.githubusercontent.com/4106760/114198293-5884ec00-997d-11eb-9e65-ce10ffb8b266.jpg)

maybe some extra coding on creality_dwin.cpp to add if not sdcard printing to Draw_Main_Menu(); when printing is complete (just an idea)

**Comment by Jyers** [comment]
I kept the confirm screen to keep it consistent with sd prints and also because I feel like its nice to see the finished print screen after a print. Functionally it does not effect anything, starting a new print will restart the print screen just fine. However, thank you for reminding me! I've been meaning to fix the 98% 00:01 thing for a while, but it completely slipped my mind. I'll make sure to get that fixed.

**Reply by aryajuanda** [reply]
thx.. 100% progress with 0 s time remaining will be match with confirmation button... thx for your great firmware....

**Reply by aryajuanda** [reply]
@Jyers .. lastest commit didn't solve the 100% progress

![IMG_3261](https://user-images.githubusercontent.com/4106760/114294216-3bf4cb00-9ac7-11eb-81d2-9b7094a1f360.jpg)
![IMG_3263](https://user-images.githubusercontent.com/4106760/114294219-3f885200-9ac7-11eb-9a90-c39bf2b9461e.jpg)

**Reply by Jyers** [reply]
Hmmmm, looks like I've made it worse. Let me see what I've messed up.

**Reply by aryajuanda** [reply]
![image](https://user-images.githubusercontent.com/4106760/115153598-15bfc400-a0a1-11eb-9b13-e5fdfbdef1c4.jpeg)

Last commit fix it... thx
